monero-outreach-docs/translations/pt-br/website-sections/your_privacy_pt_pt.md

```diff
@@ -0,0 +1,25 @@
+# A Sua Privacidade
+
+**“The caged bird sings with a fearful trill, of things unknown, but longed for still, and his tune is heard on the distant hill, for the caged bird sings of freedom.” - Maya Angelou - 4 de janeiro de 2021**
+
+Com tantos termos de privacidade escritos para confundir e desencorajar a leitura, estamos satisfeitos pela oportunidade de falar sobre a sua privacidade e a nossa responsabilidade para com ela. Afinal, isto é Monero, e devemos praticar o que defendemos e acreditamos.
+
+Resumo: O nosso site armazena o mínimo de dados possível, durante o menor período de tempo possível, da forma mais segura possível. Não partilharemos os seus dados com ninguém para fins lucrativos, mas, visto estes se encontrarem num servidor, deve considerá-los públicos.
+
+**Terceiros**  
+Os serviços web de terceiros diminuem a sua privacidade, por isso não gostamos da sua utilização. A única exceção é que usamos serviços da Mapbox, Inc. e Mapbox International, Inc. para fornecer mapas. Os nossos mapas estão desligados por defeito; se os ativar, MapBox terá os seus metadados de identificação das nossas páginas que contêm mapas. Identificaremos claramente cada página do nosso site onde um terceiro verá os seus metadados e dar-lhe-emos a oportunidade de os evitar. Se alguma vez utilizarmos outro serviço de terceiros, divulgá-lo-emos aqui.
+
+**Cookies**  
+Temos um cookie opcional que armazena uma identificação (ID) gerada pelo site. Este ID armazena as suas preferências do site no nosso servidor. É opcional, destina-se simplesmente a tornar o nosso site mais fácil de utilizar.
+
+**Rastreio/Relatórios**  
+O nosso site rastreia as visitas e armazena essa informação no nosso servidor. Não armazenamos o seu endereço IP; eliminamos sua <em>User Agent string</em> (conjunto de informações em que se encontra o sistema operativo, por exemplo) e armazenamos apenas o seu tipo de browser e o tamanho de ecrã. Estes são anonimizados e obtemos algumas estatísticas para ajudar a melhorar a nossa estratégia.
+
+**Formulário de Contacto/Contributo**  
+Os dados que nos fornece quando nos contacta através do site são armazenados no nosso servidor e eliminados logo após a mensagem ser recebida/respondida. Se tivermos que os guardar para referência, armazenaremos a mensagem offline.
```

Comment:
```suggestion
Os dados que nos fornece quando nos contata através do site são armazenados no nosso servidor e eliminados logo após a mensagem ser recebida/respondida. Se tivermos que os guardar para referência, armazenaremos a mensagem offline.
```

Reply:
Maybe we can put a "c" on the verb "contatar", I wasn't sure about. If so, you can ignore this suggestion.

Reply:
Yupe, in pt_PT we use the verb "contactar".